src/main/java/pl/umk/mat/git2befit/controller/workout/TrainingPlanController.java

```diff
@@ -72,14 +72,17 @@ public TrainingPlanController(TrainingPlanManufacture manufacture, TrainingPlanS
         }
         TrainingPlan trainingPlan = new TrainingPlan(trainingForm, trainingPlans);
 
+        trainingPlan.setTitle(trainingForm.getTrainingType());
+        List<TrainingPlan> savedTrainingPlan = null;
         if (authorizationToken != null) {
-            String email = JWTService.parseEmail(authorizationToken);
-            trainingPlanService.saveTrainingWithUserEmail(List.of(trainingPlan), email);
+            try {
+                String email = JWTService.parseEmail(authorizationToken);
+                savedTrainingPlan = trainingPlanService.saveTrainingWithUserEmail(List.of(trainingPlan), email);
+            } catch (Exception ignored) {
```

Comment:
Dlaczego nie obsługujecie wyjątku może chociaż zalogować że coś się stalo

Reply:
Ten wyjątek nie ma większego znaczenia i nie widzę sensu jego obsługi. Głównym zadaniem try-catcha jest złapanie wyjątku z ```JWTService.parseEmail```, bo jeżeli złapie zły token to wywala tam błąd.